### PR TITLE
layouts: Add announcement feature to all pages

### PR DESCRIPTION
**Pull request by Idadelveloper**
Signed-off-by: Ida Delphine <mida@unicef.org>

This tackles https://github.com/unicef/inventory/issues/41

### CODE REVIEW

**Comment by Idadelveloper** [comment]
@jwflory I think it's best we have a character limit for the news items. That is it shouldn't exceed a certain number of characters. That way, I can put in a fixed height for the alert box and it won't exceed that size. And also the page won't seem to be moving when the news items change. What do you think?

**Comment by jwflory** [comment]
Following up from stand-up today, we decided to use a 250 character limit for the alert content. This should also be documented in the README for future reference, since it will not be immediately clear that there is an upper-bound limit.

File: assets/css/style.css [hunk 1]
```diff
@@ -196,13 +225,13 @@ h4:hover a {
 }
 
 .section {
-    padding-top: 80px;
+    padding-top: calc(210px - 4vw);
```
Comment (jwflory) [inline assets/css/style.css:228]: Is this duplicated in the HTML layout? See here:

https://github.com/unicef/inventory-hugo-theme/pull/10/commits/1c3a979f7051db590ae52ee6227854e965fffbc9#diff-3edcca306992c0cab9f171119e108a629ef56d8e5310654a54d59d6a2791cc6aR2

Comment (Idadelveloper) [reply assets/css/style.css:228]: @jwflory not really.  Let me get rid off the repitition though.

**Comment by Idadelveloper** [comment]
@jwflory I made some changes

**Review by jwflory — APPROVED** [review]
This looks good to me. I'm excited to try this out in the site! 🚀 Going to merge this into the theme now. 🎬

**Pull request merged**